qemu/tests/openflow_acl_test.py

```diff
@@ -277,11 +277,32 @@ def setup_service(setup_target):
         error_context.context("Set up %s service in %s"
                               % (setup_params.get("service"), setup_target),
                               logging.info)
-        setup_func(setup_cmd, timeout=setup_timeout)
         if params.get("copy_ftp_site") and setup_target != "localhost":
-            ftp_site = os.path.join(data_dir.get_deps_dir(), params.get("copy_ftp_site"))
+            ftp_site = os.path.join(data_dir.get_deps_dir(),
+                                    params.get("copy_ftp_site"))
             ftp_dir = params.get("ftp_dir")
             setup_vm.copy_files_to(ftp_site, ftp_dir)
+        os_type = params["os_type"]
+        os_params = params.object_params(os_type)
+        access_param = os_params.object_params(setup_target)
+        if "ftp" in access_param.get("access_cmd"):
+            setup_func("rm -f %s" % params["vsftpd_conf_tmp"])
+            vsftpd_content = setup_func("cat %s" %
```

Comment:
When `setup_func` use `process.system_output()`, it might hit decode issue on python3, so better replace it to `process.getoutput()`